package.json

```diff
@@ -14,7 +14,7 @@
     "postcss": "^8.4.49",
     "postcss-cli": "^11.0.0",
     "sass": "^1.82.0",
-    "spotlight-frontend": "^4.5.0"
+    "spotlight-frontend": "^4.7.0"
```

Comment:
Shouldn't blacklight-frontend also be updated?

Reply:
Yes, thank you. Done.